server/routes/departmentForms.js

```diff
@@ -35,6 +34,10 @@ const saveDepartmentFrom = co.wrap(function* (departmentId, form) {
 router.get("/departments/:departmentId/forms", co.wrap(function* (req, res) {
     const departmentId = req.params.departmentId;
 
+    if (!permissionsUtils.isDepartmentManager(req.userDetails, departmentId)) {
```

Comment:
this get should be accessible to all users

Reply:
@may-ben-arie will be fixed